components/api/section.js

```diff
@@ -1,10 +1,15 @@
-import React from "react";
-import PropTypes from "prop-types";
-import Link from "next/link";
+import React from 'react';
+import PropTypes from 'prop-types';
+import Link from 'next/link';
+
+import globals from '../../const';
 
 const Section = ({ id, title, children }) => {
   return (
-    <section id={id}>
+    <section>
+      <a className="hidden-anchor" id={id}>
+        HEY
```

Comment:
what is the point of this ?

Reply:
With a sticky header anchors get under header. I avoid this using fake anchors. "HEY" I totaly useless here ^^my bad